wordpress/wp-content/plugins/memberful-wp/src/marketing_content.php

```diff
@@ -1,7 +1,23 @@
 <?php
 
 define( 'MEMBERFUL_MARKETING_META_KEY', 'memberful_marketing_content' );
-define( 'MEMBERFUL_OPTION_DEFAULT_MARKETING_CONTENT', 'memberful_default_marketing_content' );
+define( 'MEMBERFUL_LEGACY_DEFAULT_MARKETING_CONTENT', 'memberful_default_marketing_content' );
+
+
+
+function memberful_migrate_from_legacy_default(){
```

Comment:
Shouldn't we call this function from `memberful_wp_plugin_migrate_db()?` Currently, we don't call this function from anywhere, which means that the migration in the specification is not working.

Reply:
@mrhead  Good catch. I've fixed this with one modification -- I've added a hook to directly hook this function in to `admin_init` inside wordpress/wp-content/plugins/memberful-wp/src/marketing_content.php. 

Since there's no autoloader this saves us headaches from dependencies and `require` ordering.

Reply:
Thanks!